topics/icrc_25_signer_interaction_standard.md

```diff
@@ -213,7 +198,7 @@ Response
     "result": {
         "scopes": [
             {
-                "method": "icrc31_get_principals"
+                "method": "icrc27_get_accounts"
             },
             {
                 "method": "icrc49_call_canister",
```

Comment:
Response for granted permissions shouldn't contain any information about the sender

Reply:
If nothing else, it's a loophole in permissions requests:
- Dapp requests accounts -> user denies
- Dapp requests calling canister -> user accepts and sends a principal anyway, bypassing the denied request for accounts. Also, how would the user select their principal(s) for this permission?

Reply:
I think the original intention was to allow signers to share multiple identities but only allow transactions with a subset of these. Of course, a sensible signer implementation would then tie the restriction to the user-choice of sharing the identity in the first place (and not include it, if the identity wasn't shared).

We should probably have a discussion about the additional scope restrictions of ICRC-49.
I removed it for now.